blocks/header/header.css

```diff
@@ -1,3 +1,828 @@
 .header {
+    background-color: #fff;
+    color: #151517;
+    letter-spacing: .3em;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-family: Gotham, arial, sans-serif;
+    font-weight: 500;
+    z-index: 5000000;
+    position: sticky;
+    top: -1px
+}
+
+.header a {
+    text-decoration: none;
+}
+
+.header button {
+    background: none;
+    border: none;
+    padding: 0;
+}
+
+.header hr {
+    margin: 0;
+    border: none;
+    border-top: 1px solid #ededed;
+}
+
+.header-logo {
```

Comment:
Can you add a common class prefix to all selectors ? eg `.header rest-of-selector` so that your styles dont bleed into other block styles and vice versa.

Reply:
> Can you add a common class prefix to all selectors ? eg `.header rest-of-selector` so that your styles dont bleed into other block styles and vice versa.

Some of these already have header- prefix. I thought adding a common selector would be a little redundant. But I am adding them per your suggestion.